x-pack/plugins/beats_management/public/components/table/table_type_configs.tsx

```diff
@@ -52,6 +52,60 @@ export interface TableType {
   controlDefinitions(items: any[]): ControlDefinitions;
 }
 
+const dynamicStatuses = {
+  STARTING: {
+    color: 'success',
+    status: i18n.translate('xpack.beatsManagement.beatsTable.startingStatusLabel', {
+      defaultMessage: 'Starting',
+    }),
+    details: i18n.translate('xpack.beatsManagement.beatsTable.configStatus.startingTooltip', {
+      defaultMessage: 'This Beat is starting.',
+    }),
+  },
+  IN_PROGRESS: {
+    color: 'warning',
+    status: i18n.translate('xpack.beatsManagement.beatsTable.updatingStatusLabel', {
+      defaultMessage: 'Updating',
+    }),
+    details: i18n.translate('xpack.beatsManagement.beatsTable.configStatus.progressTooltip', {
+      defaultMessage: 'This Beat is currently reloading config from CM.',
+    }),
+  },
+  RUNNING: {
+    color: 'success',
+    status: i18n.translate('xpack.beatsManagement.beatsTable.runningStatusLabel', {
+      defaultMessage: 'Running',
+    }),
+    details: i18n.translate('xpack.beatsManagement.beatsTable.configStatus.runningTooltip', {
+      defaultMessage: 'This Beat is running without issues.',
+    }),
+  },
+  CONFIG: {
+    color: 'danger',
+    status: i18n.translate('xpack.beatsManagement.beatsTable.configErrorStatusLabel', {
+      defaultMessage: 'Config error',
+    }),
+  },
+  FAILED: {
+    color: 'danger',
+    status: i18n.translate('xpack.beatsManagement.beatsTable.failedStatusLabel', {
+      defaultMessage: 'Error',
+    }),
+    details: i18n.translate('xpack.beatsManagement.beatsTable.configStatus.errorTooltip', {
+      defaultMessage: 'There is an error on this beat, please check the logs for this host.',
+    }),
+  },
+  STOPPED: {
+    color: 'danger',
+    status: i18n.translate('xpack.beatsManagement.beatsTable.stoppedStatusLabel', {
+      defaultMessage: 'stopped',
+    }),
+    details: i18n.translate('xpack.beatsManagement.beatsTable.configStatus.errorTooltip', {
+      defaultMessage: 'There is an error on this beat, please check the logs for this host.',
```

Comment:
This is kind of nitpicky but wouldn't a better tooltip inform the user that the beat is stopped, but nothing more? It might be stopped intentionally and not due to an actual problem right?